Sources/SystemMetrics/SystemMetrics.swift

```diff
@@ -18,7 +18,7 @@ import Dispatch
 import Glibc
 #endif
 
-extension MetricsSystem {
+public extension MetricsSystem {
```

Comment:
Do we actually want this change? In NIO land we put access modifiers on computed properties / methods in extensions to avoid accidentally adding methods to the public API.

Reply:
good spot! Yes, please don't put any words in front of `extension`. That makes accidentally public things slip through code review.

Reply:
Oh yes. I ran `swiftformat .` once to test the soundness script. I guess we should edit the `.swiftformat` rules.

Reply:
Thanks @glbrntt very good spot!

Reply:
man, swiftformat makes so many bad edits by default